src/datasets/utils/readme.py

```diff
@@ -89,6 +86,12 @@ def parse(self):
                     if self.text != "" and self.text not in FILLER_TEXT:
                         self.is_empty_text = False
 
+        if self.level == "" and not suppress_parsing_errors:
+            if self.parsing_error_list != [] or self.parsing_warning_list != []:
+                errors = "\n".join(list(map(lambda x: "-\t" + x, self.parsing_error_list + self.parsing_warning_list)))
```

Comment:
you can use a comprehension list
```suggestion
                errors = "\n".join("-\t" + x for x in self.parsing_error_list + self.parsing_warning_list)
```

Reply:
Fixed.